src/commands/InitCommand.ts

```diff
@@ -70,6 +76,12 @@ export class InitCommand implements yargs.CommandModule {
                 console.log(chalk.green(`Project created inside current directory.`));
             }
 
+            if (args.pm && installNpm) {
+                InitCommand.executeCommand("npm install");
+            } else {
+                InitCommand.executeCommand("yarn install");
```

Comment:
Shouldn't you `await` this?  Or we don't wait for them to complete ever?

Reply:
my bad. `await` should be there.